docs/fsharp-cheatsheet.md

```diff
@@ -555,136 +555,158 @@ Another way of implementing interfaces is to use *object expressions*.
-| Module                                                            | `module internal MyModule =`               |
-| Module .. `let`                                                   | `let private value =`                      |
-| Record                                                            | `type internal MyRecord = { id: int }`     |
-| Record [ctor](#CodeOrganization_PrivateConstructors)              | `type MyRecord = private { id: int }`      |
-| Discriminated Union                                               | `type private MyDiscUni = A \| B`          |
-| Discriminated Union [ctor](#CodeOrganization_PrivateConstructors) | `type MyDiscUni = internal A \| B `        |
-| Class                                                             | `type internal MyClass() =`                |
-| Class [ctor](#CodeOrganization_PrivateConstructors)               | `type MyClass private () =`                |
-| Class Additional [ctor](#CodeOrganization_PrivateConstructors)    | `internal new() = MyClass("defaultValue")` |
-| Class .. `let`                                                    | *Always private. Cannot be modified*       |
-| `type` .. `member`                                                | `member private _.classMember =`           |
-
-<a name="CodeOrganization_PrivateConstructors"></a>
-##### Private Constructors
-
-Limiting `type` constructors (ctor) accessibility is a good way to enforce value integrity.
-
-Example of Single-case Discriminated Union with a `private` constructor:
+| Element                                                         | Example with Modifier                      |
+|-----------------------------------------------------------------|--------------------------------------------|
+| Module                                                          | `module internal MyModule =`               |
+| Module .. `let`                                                 | `let private value =`                      |
+| Record                                                          | `type internal MyRecord = { id: int }`     |
+| Record [ctor](#CodeOrganization_SmartConstructors)              | `type MyRecord = private { id: int }`      |
+| Discriminated Union                                             | `type internal MyDiscUni = A \| B`         |
+| Discriminated Union [ctor](#CodeOrganization_SmartConstructors) | `type MyDiscUni = private A \| B `         |
+| Class                                                           | `type internal MyClass() =`                |
+| Class [ctor](#CodeOrganization_SmartConstructors)               | `type MyClass private () =`                |
+| Class Additional [ctor](#CodeOrganization_SmartConstructors)    | `internal new() = MyClass("defaultValue")` |
+| Class .. `let`                                                  | *Always private. Cannot be modified*       |
```

Comment:
```suggestion
| Class .. `let`                                                  | *Always private. Cannot be overridden*       |
```
(I would want to slip in a reference to `member val`, but maybe best to leave it!)

Reply:
Yeah, wasn't sure how far to take it. `val`s feel dirty.